playbooks/open-sourcing.md

```diff
@@ -0,0 +1,90 @@
+
+- [ ] Use Github to search for telltale strings such as `key`, `token`, `secret`, `password`, `credential`
+- [ ] On the results page, check both:
+  - [ ] the **Code** tab, to view results from the current state of the repo
+  - [ ] the **Commits** tab, to view historical commits that might contain secrets that would become newly visible
+        to the public
+
+#### Automated scan
+
+As an extra precaution, check the current state of your repo with a tool such as
+https://github.com/Yelp/detect-secrets. This is a straightforward utility which scans a codebase for known secret
+patterns such as AWS keys, as well as "high entropy strings" that are typical of keys and tokens in general.
+
+- [ ] Use a tool such as detect-secrets to:
+  - [ ] Perform an initial scan and audit any found secrets
+  - [ ] Consider adding a git hook to prevent committing secrets in the future
+
+## Review other repository areas for sensitive content
+
+It is possible that the repo's issue backlog, or wiki, or "projects" (the kanban style boards that GH repos can
+contain) might contain discussion of sensitive matters, so it is a good idea to review all of these as well.
+
+- [ ] Review open _and_ closed issues
+- [ ] Review "projects"
+- [ ] Review wiki content
+
+## Modify CI settings
+
+We have two ways of accepting contributions on our projects:
+
+1.  PRs are contributed via branches on the main Artsy repo, versus…
+2.  PRs are contributed via branches on _forked_ repos
+
+For apps that need to have secrets at build time (such as deploy keys, or GitHub tokens for private Ruby gems,
+etc), option #2 represents a security risk, as any user can submit a pull request and potentially have your
+project's secrets injected into their modified version of your app.
+
+To ensure safety on CI you should review the following settings under Project Settings > Build Settings > Advanced
+Settings:
+
+- [ ] **Pass secrets to builds from forked pull requests** should _definitely_ be set to **Off**
+- [ ] **Build forked pull requests** should be _probably_ set to **Off** (if it is not possible to build a project
+      at all without the injected secrets)
+
+If both of these are set to off, that implies that new contributions will have to come from branches on the main
+Artsy repo, so you might also need to
+
+- [ ] Update the project's README to describe the new required contribution style
+
+## Spread the word
+
+- [ ] Consider adding a post to https://github.com/artsy/artsy.github.io
+
+Now that your project is public, you might want to write a blog post to document it, for the community as well as
+for future Artsy developers.
+
+Some previous examples:
+
+- [On our implementation of React Native](https://artsy.github.io/blog/2016/08/24/On-Emission/) — Emission
+- [Modernizing Force](https://artsy.github.io/blog/2017/09/05/Modernizing-Force/) — Stitch / express-reloadable
+- [Apogee: Doing More with Less](https://artsy.github.io/blog/2018/02/02/artsy-apogee/) — Apogee
```

Comment:
Thanks for the apogee callout! Unfortunately, it was never OSS'd since it exposed our Auction partners' data structures. I think we could include the following, though:

- https://artsy.github.io/blog/2015/04/28/how-we-open-sourced-eigen/
- https://artsy.github.io/blog/2015/08/06/open-sourcing-energy/